src/media/css/elements--header.styl

```diff
@@ -130,14 +130,25 @@ mkt-header-child {
     border: 0;
     border-radius: 5px;
     height: 70px;
-    padding: 15px 0 15px 20px;
-    position: relative;
     outline: none;
+    padding: 15px 40px;
```

Comment:
Is the 40px for the icon? That might go into header--search since it's not part of the core element.